### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tpatel**
Removing the "Only contribute to one side of the project" part of the docs as it's too complex vs the value it provides (no regular contributor have been using it).

### CODE REVIEW

**Comment by ramnes** [comment]
And actually I just used that part of the guide a few minutes ago and was happy to find it. 😬

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -118,36 +115,3 @@ Once you create a pull request, the tests will automatically run. It is a good p
 
 1. Find the folder containing the e2e test that you're looking for in `cypress/e2e`.
```
[the remark this answers is not in the document]
Comment (tpatel) [reply .github/CONTRIBUTING.md:116]: I believe #148 only touches line 120, which I'm not touching

[the remark this answers is not in the document]
Comment (tpatel) [reply .github/CONTRIBUTING.md:113]: My bad, git did give me a conflict, anyway it's fixed

**Comment by tpatel** [comment]
@ramnes could you share what you've used? frontend-only or backend-only?

**Comment by ramnes** [comment]
backend-only

**Comment by tpatel** [comment]
Was it ok to have a new page open each time you save a file?

**Comment by tpatel** [comment]
After talking with @ramnes it looks like this section is useful, so this PR doesn't make sense anymore.

**Pull request closed without merging**